R/derive_params_growth_height.R

```diff
@@ -68,7 +68,13 @@
 #'
 #' @param analysis_var Variable containing anthropometric measurement
 #'
-#' A numeric vector is expected, e.g. `AVAL`, `VSSTRESN`
+#'  A numeric vector is expected, e.g. `AVAL`, `VSSTRESN`
+#'
+#' @param who_correction Right skew correction
+#'
+#'  A logical scalar, e.g. `TRUE`/`FALSE` is expected.
+#'  WHO developed a modification to the z-score calculation to accommodate for right-skewness
+#'  in certain data, if set to `TRUE` the WHO correction is applied.
```

Comment:
See comment above

Reply:
Zelos - i'll make a commit to update this and please check you agree

Reply:
@rossfarrugia @zdz2101 @milei.  About WHO BMI SDS and pctls.  WHO does calculate these, but almost all people use weight-for-length rather than BMI among <2s.  The only times I've seen BMI used among infants is when someone wants to see if BMI SDS is a better predictor of subsequent obesity than is WFL SDs, and the results are not strong enough to change to BMI SDS.  The only advantage of BMI SDS that I know of is that it isn't limited by the length of the infants.  For WFL SDS, if you've got a very short or very tall child, the length will fall outside the WHO length limits and  WFL z/pctl will be NA.  If it isn't difficult to add BMI SDS/pctl for infants to the program, go ahead, but honestly, I don't think it's a priority because it's so little used.

Reply:
@Ross - sorry, I had no idea that Roche wanted this.  I think this is a really strange request, but glad it won't be extra work.

Reply:
no problem! i might query it with them later when i come to build the Roche specific template based on admiralpeds, as there's a lot of other things missed too from their standards like the WHO skewness piece you pointed out to us.